vhdl/subsystems/vnir/LVDS_reader/lvds_reader_v3.qpf

```diff
@@ -0,0 +1,31 @@
+# -------------------------------------------------------------------------- #
```

Comment:
Are these project files necessary to the rest of your commit? If not it might be a good idea to remove them from the pr, because we'll be remaking them when we do the FPGA subsystem anyway

Reply:
No they aren't necessary.  I just uploaded the entire project as a complete package so that people would be able to test it without too much hassle.  As I had mentioned previously, some minor modifications are necessary to include the LVDS reader into a new project (I'm not sure if IPs can just be transferred between different projects of if they need to be instantiated with the Wizard again)

Reply:
IPs can be transferred fine as far as I know, because all the wizard does is provide a thin wrapper around built-in components.